duo_wordpress.php

```diff
@@ -468,6 +485,13 @@ function duo_update_mu_options() {
         else {
             $result = update_site_option('duo_xmlrpc', 'on');
         }
+        if(isset($_POST['duo_rest'])) {
+            $rest = sanitize_text_field($_POST['duo_rest']);
+            $result = update_site_option('duo_rest', $rest);
```

Comment:
@JohnMaguire I understand that. My point is that the code you are talking about resides in the `duo_update_mu_options` function. The `duo_update_mu_options` is hooked in as a part of ` add_action('update_wpmu_options', 'duo_update_mu_options');`

I never said it was an issue with MU, but the point you are making is regarding code that would technically only run on MU.

I understand the fix you want to make and I could add it in, but it's a moot point since DUO does not apparently work on MU.